pyecharts/option.py

```diff
@@ -790,6 +790,11 @@ def datazoom(is_datazoom_show=False,
         "xAxisIndex": datazoom_xaxis_index,
         "yAxisIndex": datazoom_yaxis_index
     }
+    if datazoom_type == "both":
+        _datazoom["type"] = "slider"
+        _datazoom_inside = _datazoom.copy()
+        _datazoom_inside["type"] = "inside"
+        return [_datazoom, _datazoom_inside]
```

Comment:
Could you please refactor the return statements in the context of this function? Better to have a single return function in one function.